rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeStorageProviderImpl.java

```diff
@@ -91,4 +100,26 @@ public Optional<Coin> getLockingCap(ForBlock activations) {
             ))
         );
     }
+
+    @Override
+    public void setWeiTransferredToUnionBridge(co.rsk.core.Coin weiTransferred) {
+        if (weiTransferred != null && weiTransferred.compareTo(co.rsk.core.Coin.ZERO) < 0) {
```

Comment:
```suggestion
        if (nonNull(weiTransferred) && weiTransferred.compareTo(co.rsk.core.Coin.ZERO) < 0) {
```

Since you are already using that package in this file.